### PR TITLE
Add --sort-keys/-S option to _jq

### PR DESCRIPTION
**Pull request by gkze**
Also sorted the other options

### CODE REVIEW

**Comment by hhatto** [comment]
@gkze 

Thank you for contribution.

Can you add also `--arg-file`, `--unbufferd` and `-e,--exit-status` options?
https://raw.githubusercontent.com/stedolan/jq/master/NEWS

**Comment by gkze** [comment]
@hhatto Sure thing! There you go

File: src/_jq [hunk 1]
```diff
@@ -40,17 +40,22 @@
...
-  '(-a --ascii-output)'{-a,--ascii-output}'[output with pure ASCII character]'
-  '(-r --raw-output)'{-r,--raw-output}'[not format string result as a JSON string with quotes]'
-  '--arg[passes a value to the jq program, e.g. --arg foo bar]:jq variable: '
+  '(--argfile)'{--argfile}'[This option passes the first value from the named file as a value to the jq program as a predefined variable]'
+  '(--unbuffered)'{--unbuffered}'[Flush the output after each JSON object is printed]'
+  '(-C --color-output)'{-C,--color-output}'[Colorize the output even if writing to a pipe or a file]'
+  '(-I --online-input)'{-I,--online-input}'[When the top-level input value is an array produce its elements instead of the array]'
+  '(-M --monochrome-output)'{-M,--monochrome-output}'[Not colorize the output]'
+  '(-R --raw-input)'{-R,--raw-input}'[Parse the input as not JSON but string]'
+  '(-S --sort-keys)'{-S, --sort-keys}'[Output the fields of each object with the keys in sorted order]'
```
Comment (hhatto) [inline src/_jq]: `'{-S, --sort-keys}'` not need brank, correctly is `'{-S,--sort-keys}'` .

File: src/_jq [hunk 2]
```diff
@@ -40,17 +40,22 @@
...
-  '(-R --raw-input)'{-R,--raw-input}'[parse the input as not JSON but string]'
-  '(-n --null-input)'{-n,--null-input}'[run the filter using null as the input]'
-  '(-c --compact-output)'{-c,--compact-output}'[compact output]' \
-  '(-C --color-output)'{-C,--color-output}'[colorize the output even if writing to a pipe or a file]'
-  '(-M --monochrome-output)'{-M,--monochrome-output}'[not colorize the output]'
-  '(-a --ascii-output)'{-a,--ascii-output}'[output with pure ASCII character]'
-  '(-r --raw-output)'{-r,--raw-output}'[not format string result as a JSON string with quotes]'
-  '--arg[passes a value to the jq program, e.g. --arg foo bar]:jq variable: '
+  '(--argfile)'{--argfile}'[This option passes the first value from the named file as a value to the jq program as a predefined variable]'
+  '(--unbuffered)'{--unbuffered}'[Flush the output after each JSON object is printed]'
```
Comment (hhatto) [inline src/_jq]: if the argument is one (`--argfile` and `--unbuffered`), correctly is `'--argfile[This option ...]'` .

**Comment by gkze** [comment]
Ah gotcha, thanks. Pushed

File: src/_jq [hunk 3]
```diff
@@ -40,17 +40,22 @@
...
-  '(-R --raw-input)'{-R,--raw-input}'[parse the input as not JSON but string]'
-  '(-n --null-input)'{-n,--null-input}'[run the filter using null as the input]'
-  '(-c --compact-output)'{-c,--compact-output}'[compact output]' \
-  '(-C --color-output)'{-C,--color-output}'[colorize the output even if writing to a pipe or a file]'
-  '(-M --monochrome-output)'{-M,--monochrome-output}'[not colorize the output]'
-  '(-a --ascii-output)'{-a,--ascii-output}'[output with pure ASCII character]'
-  '(-r --raw-output)'{-r,--raw-output}'[not format string result as a JSON string with quotes]'
-  '--arg[passes a value to the jq program, e.g. --arg foo bar]:jq variable: '
+  '--argfile[This option passes the first value from the named file as a value to the jq program as a predefined variable]'
+  '--unbuffered)[Flush the output after each JSON object is printed]'
```
Comment (hhatto) [inline src/_jq]: `--unbuffered)` not need `)`
Merge After you fix this :-)

**Comment by gkze** [comment]
Sorry about that, that was rushed :)

**Comment by hhatto** [comment]
LGTM

Thanks!!

**Pull request merged**